openapi-v3.yaml

```diff
@@ -1388,31 +1493,36 @@ components:
           format: "date-time"
         owner_thread_id:
           type: "string"
+          nullable: true
         executed_by_thread_id:
           type: "string"
         artifacts_saved:
           type: "boolean"
-        artifacts_s3_path:
+        artifact_s3_path:
```

Comment:
Was this an error in our documentation? I am assuming the actual endpoint uses `artifact_s3_path`?

Reply:
Same question, want to make sure this is intentional!

Reply:
yep! that's what the actual endpoint returns